boards/same54-xpro/include/periph_conf.h

```diff
@@ -406,6 +406,18 @@ static const sam0_common_gmac_config_t sam_gmac_config[] = {
 };
 /** @} */
 
+/**
+ * @name FREQM peripheral configuration
+ * @{
+ */
+static const freqm_config_t config_freqm[] = {
```

Comment:
```suggestion
static const freqm_config_t freqm_config[] = {
```

for consistency 

then you can add it to `dist/tools/doccheck/generic_exclude_patterns` :sweat_smile: